### DISCUSSION TITLE
An Overview of the Release Numbering Scheme

### DISCUSSION THREAD

**Discussion by frenchie68** [frenchie68/Z79Forth#16] [Announcements]
Github does not offer the possibility of adding a README to the releases page so I will have that here in the hope that it is read by interested people.

- 3.x releases: based on the REL-ANS94 branch. Interrupt driven serial console. Actively maintained.
- 2.x releases: based on the master branch. 79-STANDARD is the primary allegiance. Includes FORTH-83 words and ANSI extensions. Interrupt driven serial console. Actively maintained.
- 1.x releases: based on the PollingArchitecture branch. 79-STANDARD is the primary allegiance. Includes FORTH-83 words and ANSI extensions. Interrupt driven serial console. No longer maintained.

To sum it up, you should go for the latest 3.x release if you want ANS94 Core, or the latest 2.x release if you want the historic 79-STANDARD flavour of Z79Forth. As of late, the latest improvements come from the ANS94 branch and will eventually be back ported to the master branch if they are relevant.

It should be noted that both the 3.x and 2.x releases include a subset of the Forth2012 test suite. However, what is actually been tested differs. Also the supplied CompactFlash images found in SW/util/*.img are different.
